touchforms/formplayer/api.py

```diff
@@ -41,16 +41,22 @@ def __init__(self, username, password):
     def to_dict(self):
         return {'type': self.type, 'key': self.key, 'username': self.username}
 
-            
-class TouchformsException(Exception):
```

Comment:
Nothing in the codebase catches the old `TouchformsException`, so replacing it with `TouchformsError`